src/main/resources/Diagram/DiagramEditSheet.xml

```diff
@@ -767,6 +774,20 @@ define('diagramStore', ['jquery', 'xwiki-meta', 'xwiki-utils', 'draw.io', 'xwiki
     oldImportFiles.apply(this, newArgs);
   }
 
+  var originalLoadUrl = EditorUi.prototype.loadUrl;
+  EditorUi.prototype.loadUrl = function(url, success, error, forceBinary, retry, dataUriPrefix) {
+    let exportedUrl = diagramUtils.getParameterValueFromURL('url', url);
+    if (exportedUrl) {
+      let decodedUrl = decodeURIComponent(exportedUrl);
```

Comment:
Why do you need to decode the parameter value? ``getParameterValueFromURL`` does it already.